### ISSUE TITLE
How to apply mods

### ISSUE THREAD

**Issue by Kelvfimer** [yuzu-emu/yuzu-android#6]
There is no way to apply pchtxt mods on android.

**Comment by MonsterDruide1** [comment]
Well yes, there is - from the settings menu, press "Open Yuzu folder". Here, you can load mods into the "load" directory, just like on PC.

**Comment by Kelvfimer** [comment]
Sure but how i can select the mod. The files are on the load/game id folder but I can't select the mod in the main menu as in the yuzu pc

**Comment by Kelvfimer** [comment]
Btw i forgot to say that also crushes yuzu early adopter (yellow one) i'm trying to apply remove blur mod on Zelda awakening.

**Comment by Kelvfimer** [comment]
I think developer needs to change the app to let yuzu access all files. Now it only grants access to a single game folder at first execution. For instance Citra has global files access.

**Comment by Kelvfimer** [comment]
As a workaround if you copy the content of the mod zip file as is in load/game id it works.if you create the folder structure with the mod name/exefs and copy only the mod pchtxt files it ends in the acess denied.

I think can be interesting to develop the mod selection menu.

**Comment by MonsterDruide1** [comment]
What's the difference between the folder structures you get in each scenario? Does your mod contain more files than just the `pchtxt`s?

**Comment by MonsterDruide1** [comment]
> For instance Citra has global files access.

Android wants to get rid of apps being able to access all files, and instead forces all newer apps to get a scoped permission instead. That's why you manually have to select the games folder.

**Comment by Kelvfimer** [comment]
> What's the difference between the folder structures you get in each scenario? Does your mod contain more files than just the `pchtxt`s?

Nope, new folders have same structure and names as the one in the zip. I think it is the permissions of the folders that I created should be different as the one in the zip. However I don't know how to change them in Android without root.
